proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -17,6 +20,63 @@ service Service {
     rpc GetTx(GetTxRequest) returns (GetTxResponse) {
         option (google.api.http).get = "/cosmos/tx/v1beta1/tx/{hash}";
     }
+
+    // TxsByEvents fetches txs by event.
+    rpc TxsByEvents(GetTxsEventRequest) returns (TxsByEventsResponse) {
```

Comment:
Inconsistency between `TxsByEvents` (plural) and `GetTxsEventRequest` (singular). I prefer to singular version.